quantuminspire/util/api/remote_backend.py

```diff
@@ -86,6 +88,7 @@ async def _create_flow(
     ) -> int:
         """Call the necessary methods in the correct order, with the correct parameters."""
         async with ApiClient(self._configuration) as api_client:
+            await self._set_team_member_id(api_client)
```

Comment:
This is only when running an algorithm, not when you log in, right?

Reply:
Moved it to login logic